apps/keyboard/js/imes/latin/predictions.js

```diff
@@ -62,13 +63,13 @@ var Predictions = function() {
   var CACHE_SIZE = 255;    // how many suggestions to remember
 
   // Weights of various permutations we do when matching input
-  var VARIANT_FORM_MULTIPLIER = .99;           // slightly prefer exact match
-  var PUNCTUATION_INSERTION_MULTIPLIER = .95;  // apostrophes are almost free
+  var VARIANT_FORM_MULTIPLIER = 0.99;           // slightly prefer exact match
```

Comment:
add leading 0 since in jsperf there's no obvious winner http://jsperf.com/dot-leading-float-vs-normal-float/2